stores/user_store.jsx

```diff
@@ -459,6 +459,22 @@ class UserStoreClass extends EventEmitter {
         return this.getStatuses()[id] || UserStatuses.OFFLINE;
     }
 
+    getTimezone(id) {
+        const profile = store.getState().entities.users.profiles[id];
+        if (profile && profile.timezone) {
+            return {
+                ...profile.timezone,
+                useAutomaticTimezone: profile.timezone.useAutomaticTimezone === 'true',
```

Comment:
I mentioned this above, but why switch types here?

Reply:
@hmhealey this is a good point to address the convention MM uses for storing booleans in postgres.

From my investigation, MM has been storing booleans as strings. See: https://github.com/mattermost/mattermost-webapp/blob/master/components/user_settings/notifications/user_settings_notifications.jsx#L138

Looking at the database this is what the notify_props looks like:
```
{ 
   "channel":"true",
   "comments":"never",
   "desktop":"mention",
   "desktop_duration":"5",
   "desktop_sound":"true",
   "email":"true",
   "first_name":"false",
   "mention_keys":"admin",
   "push":"mention",
   "push_status":"away"
}
```

I'm following the same convention MM has been using for storing booleans.  If this is not the preferred convention, let us know what's the best way to store booleans.